scripts/parse_live.py

```diff
@@ -72,29 +73,34 @@ def section_name_to_anchor(section):
     section = section.replace('%', '.')
     return section
 
-def query_pageids(wiki, pageids):
+def query_pageids(wiki, pageids, revids):
```

Comment:
Maybe rename to `query_article_data` or something like that, now that we don't require pageids?

Reply:
Yeah, I'll rename it. What do you mean by "we don't require pageids"? But we'll need pageids when importing from the pageid file?

Reply:
I mean, this function does not require the caller to pass pageids, because they can pass just revids. So it's a little odd to have pageids in the name.

Reply:
I see, thank you for the explanation! :)